assets/javascripts/discourse/initializers/assign-user-menu.js

```diff
@@ -0,0 +1,40 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "assign-user-menu",
+
+  initialize(container) {
+    withPluginApi("1.2.0", (api) => {
+      if (api.registerUserMenuTab) {
+        const siteSettings = container.lookup("service:site-settings");
+        if (!siteSettings.assign_enabled) {
+          return;
+        }
+
+        const currentUser = api.getCurrentUser();
+        if (!currentUser?.can_assign) {
+          return;
+        }
+        api.registerUserMenuTab((UserMenuTab) => {
+          return class extends UserMenuTab {
+            get id() {
+              return "assign-notifications";
```

Comment:
```suggestion
              return "assign-list";
```

The list is not all notifications so using `notifications` here doesn't feel right.